fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -1965,7 +1965,7 @@ class FMCMainDisplay extends BaseAirliners {
 
     set cruiseFlightLevel(fl) {
         this._activeCruiseFlightLevel = Math.round(fl);
-        SimVar.SetSimVarValue("L:AIRLINER_CRUISE_ALTITUDE", "number", this._activeCruiseFlightLevel * 100);
```

Comment:
L:AIRLINER_CRUISE_ALTITUDE -> L:A32NX_AIRLINER_TO_FLEX_TEMP

We may want to keep this L:Var change in mind for supporting legacy external applications

Reply:
I can add a note to the PR description - but as this LVar is not even documented in the fbw-a32nx/docs/a320-simvars.md file I think we are fine. 
I'll add these to the a320-simvars,md though.